### PR TITLE
[0.3.x] CAL-396 Upgraded to dependency-check-maven 3.0.2

### PR DESCRIPTION
**Pull request by mackncheesiest**
#### What does this PR do?
Upgrades dependency-check-maven plugin to 3.0.2

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)
@brjeter @clockard  

#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)
@bdeining @brendan-hofmann

#### How should this be tested?
CI passes including OWASP execution 
#### What are the relevant tickets?
[CAL-396](https://codice.atlassian.net/browse/CAL-396)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by mackncheesiest** [comment]
build now

File: pom.xml [hunk 1]
```diff
@@ -632,12 +632,20 @@
...
                         <artifactId>dependency-check-maven</artifactId>
-                        <version>1.4.5</version>
+                        <version>3.0.2</version>
                         <configuration>
                             <failBuildOnCVSS>7</failBuildOnCVSS>
                             <format>ALL</format>
-                            <suppressionFile>dependency-check-maven-config.xml</suppressionFile>
+                            <suppressionFiles>
+                                <suppressionFile>https://raw.githubusercontent.com/codice/ddf/master/dependency-check-maven-config.xml</suppressionFile>
+                                <suppressionFile>https://raw.githubusercontent.com/codice/ddf-support/master/support-owasp/src/main/resources/dependency-check-maven-config.xml</suppressionFile>
```
Comment (clockard) [inline pom.xml]: Do we want to include this suppression file? I don't think it is maintained anymore.

Comment (mackncheesiest) [reply pom.xml]: I wasn't sure if it was being maintained or not. I can remove it.

**Comment by mackncheesiest** [comment]
build now

**Comment by brendan-hofmann** [comment]
Hm, looks like the build failed on OWASP... 🤔

**Comment by mackncheesiest** [comment]
It passed locally last I tried 😢 
I'll look into it

**Comment by mackncheesiest** [comment]
build now

**Pull request merged**